sys/dev/athn/if_athn_usb.c

```diff
@@ -84,6 +84,23 @@
 // map OpenBSD flag name to FreeBSD
 #define	IFF_RUNNING	IFF_DRV_RUNNING
 
+// missing functions to time conversion
```

Comment:
It's a huge piece of new code. I'd rather see it in a new file than written directly inside if_athn_usb.c.
Can be a header file as we're dealing with inline functions here only.